Is the "engines" field in package.json correct?

`node 0.10.x` is the current value: https://github.com/ben-ng/sourcemap-validator/blob/master/package.json#L41

Running a `npm install` on a project, I receive this error:
```
npm ERR! Darwin 16.0.0
npm ERR! argv "/Users/arnaudbesnier/.nvm/versions/node/v7.10.0/bin/node" "/Users/arnaudbesnier/.nvm/versions/node/v7.10.0/bin/npm" "i"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! code ENOTSUP

npm ERR! notsup Unsupported engine for sourcemap-validator@1.0.5: wanted: {"node":"0.10.x"} (current: {"node":"7.10.0","npm":"4.2.0"})
npm ERR! notsup Not compatible with your version of node/npm: sourcemap-validator@1.0.5
npm ERR! notsup Not compatible with your version of node/npm: sourcemap-validator@1.0.5
npm ERR! notsup Required: {"node":"0.10.x"}
npm ERR! notsup Actual:   {"npm":"4.2.0","node":"7.10.0"}

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/arnaudbesnier/.npm/_logs/2017-08-23T15_56_19_664Z-debug.log
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,6 +6,7 @@ node_js:
   - "4"
   - "6"
   - "7"
+  - "8"
 
 before_install:
   - npm config --global set spin false

--- a/package.json
+++ b/package.json
@@ -38,6 +38,6 @@
   },
   "optionalDependencies": {},
   "engines": {
-    "node": "0.10.x"
+    "node": "^0.10 || ^4.5 || 6.* || >= 7.*"
   }
 }

